src/app/leverage/components/stats.tsx

```diff
@@ -29,11 +26,11 @@ export function Stats() {
       <div className='bg-ic-gray-800 h-full w-[1px]' />
       <StatsItem
         token={indexToken}
-        change24h={''}
-        change24hIsPositive={true}
+        change24h={navchange !== 0 ? formatPercentage(navchange) : ''}
+        change24hIsPositive={navchange > 0}
         high24h={high24h}
         low24h={low24h}
-        price={indexTokenPrice}
+        price={nav > 0 ? formatDollarAmount(nav) : ''}
```

Comment:
minor: all these changes could be moved into the formatting hook which would make this look a lil cleaner

Reply:
@janndriessen good call - i'll update this one and other usages in a follow-up PR